### PR TITLE
Improve CSS JS minification libraries

### PR DESCRIPTION
**Pull request by timotei-litespeed**
Used library for CSS and JS minification: https://github.com/matthiasmullie/minify with small changes to use use our namespace

### CODE REVIEW

File: lib/css_js_min/minify/minify.cls.php [hunk 1]
```diff
@@ -0,0 +1,550 @@
...
+ * 
+ * @author Matthias Mullie <minify@mullie.eu>
+ * @copyright Copyright (c) 2012, Matthias Mullie. All rights reserved
+ * @license MIT License
+ */
+
+namespace LiteSpeed\Lib\CSS_JS_MIN\Minify;
+
+use LiteSpeed\Lib\CSS_JS_MIN\Minify\Exception\IOException;
+use Psr\Cache\CacheItemInterface;
```
Comment (hi-hai) [inline lib/css_js_min/minify/minify.cls.php:13]: @timotei-litespeed Where is this class from?

Comment (timotei-litespeed) [reply lib/css_js_min/minify/minify.cls.php]: Removed the class and the function that used it

File: src/optimizer.cls.php [hunk 2]
```diff
@@ -136,6 +136,16 @@ public function serve($request_url, $file_type, $minify, $src_list)
 			File::save($tmp_static_file, $content, true, true);
 		}
 
+		// if CSS - run the minification on the saved file.
+		// Will move imports to the top of file and remove extra spaces.
+		if ($file_type == 'css') { 
+			$obj = new Lib\CSS_JS_MIN\Minify\CSS();
+			$obj->add(File::read($tmp_static_file));
+			$css = $obj->minify();
```
Comment (hi-hai) [inline src/optimizer.cls.php:144]: Here you always minify the file regardless user settings? @timotei-litespeed

Comment (timotei-litespeed) [reply src/optimizer.cls.php]: Fixed in last push

**Comment by hi-hai** [comment]
@timotei-litespeed plz update the desc to mention what is fixed

**Pull request merged**